lib/install/management/upgrade.go

```diff
@@ -158,15 +187,15 @@ func (d *Dispatcher) deleteSnapshot(snapshotName string, applianceName string) e
 func (d *Dispatcher) tryCreateSnapshot(name, desc string) error {
 	defer trace.End(trace.Begin(name))
 
-	upgrading, snapshot, err := d.appliance.UpgradeInProgress(d.ctx, UpgradePrefix)
-	if err != nil {
-		return err
-	}
-	if upgrading {
-		return errors.Errorf("Detected another upgrade process in progress. If this is incorrect, manually remove appliance snapshot %q and restart upgrade", snapshot)
-	}
+	// upgrading, snapshot, err := d.appliance.UpgradeInProgress(d.ctx, UpgradePrefix)
```

Comment:
yeah I won't leave it like this. This is primarily what I was referring to in the description regarding cleanup